Dockerfile

```diff
@@ -0,0 +1,45 @@
+# Dockerfile
+
+FROM phusion/passenger-ruby22:0.9.15
+
+# Set correct environment variables.
+ENV HOME /root
+ENV RAILS_ENV production
+
+# Use baseimage-docker's init process.
+CMD ["/sbin/my_init"]
+
+# Expose Nginx HTTP service
+EXPOSE 80
```

Comment:
Yep, this was based on my misunderstanding that Elastic Beanstalk always set up an ELB.